src/systempolicy/systemPolicy.go

```diff
@@ -952,6 +954,7 @@ func PopulateSystemPoliciesFromSystemLogs(sysLogs []types.KnoxSystemLog) []types
 			}
 
 			polCnt := 0
+			isWpfsDbUpdated = false
```

Comment:
I suggest we can remove `isWpfsUpdated := false` from line 923

Reply:
This was a bug I fixed. Earlier if you have logs from multiple clusters .. this flag was not getting reset for every cluster.. Thus the policies were regenerated regardless of any updates to the DB.

Reply:
Fixed.